config/manager/env.yaml

```diff
@@ -16,4 +16,7 @@ spec:
             - name: LINUXPTP_DAEMON_IMAGE
               value: "quay.io/openshift/origin-ptp:4.9"
             - name: KUBE_RBAC_PROXY_IMAGE
-              value: "quay.io/openshift/origin-kube-rbac-proxy:4.9"
\ No newline at end of file
+              value: "quay.io/openshift/origin-kube-rbac-proxy:4.9"
+            - name: SIDECAR_EVENT_IMAGE
+              value: "quay.io/openshift/cloud-event-proxy:4.9"
```

Comment:
I am not able to pull this image

podman pull quay.io/openshift/cloud-event-proxy:4.9
Error response from daemon: unauthorized: access to the requested resource is not authorized

Reply:
it's not yet ready. Nishant is working on building this image, right now it's in a private repo. This PR cannot be merged till that repo is ready